project/plugins.sbt

```diff
@@ -2,14 +2,14 @@ addSbtPlugin("com.cavorite" % "sbt-avro-1-8" % "1.1.3")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
 addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.4.1")
 addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.6.3")
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.7")
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.8")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
 addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.2")
 addSbtPlugin("org.scalastyle" % "scalastyle-sbt-plugin" % "1.0.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.0")
-addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.27")
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.3")
 
 libraryDependencies ++= Seq(
-  "com.github.os72" % "protoc-jar" % "3.3.0.1"
+  "com.github.os72" % "protoc-jar" % "3.5.1.1"
```

Comment:
This may pull in a newer `protoc` which might break the `protobuf-java` dependency we have?
Normally we want it to have the same `protoc` version as our `protobuf-java` dep.

Reply:
@nevillelyh good point

Reply:
@nevillelyh updated .. also made a PR to `protobuf-generic`

Reply:
I would be cautious about updating protos, actually would prefer to keep it at the same level as Beam.